scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
+
+DATA_TYPE = ['png','PNG','jpg','JPG']
```

Comment:
Blank line, can be automatically fixed by `python-black`

Reply:
Fixed with `python-black`